mrp_production_force_bom/models/mrp_production.py

```diff
@@ -28,7 +28,13 @@ def _skip_bom_check(self):
 
     def _check_component_discrepancies(self, error_messages):
         # Get all the components and their quantities using explode()
-        _, lines_done = self.bom_id.explode(self.product_id, self.product_qty)
+        factor = (
```

Comment:
`factor` sounds a bit strange to me.  Why don't we follow the same style as the standard code? https://github.com/odoo/odoo/blob/c1cd95b/addons/mrp/models/mrp_production.py#L869-L870

Reply:
Just ignore my above comment.

c.f. https://github.com/odoo/odoo/blob/c1cd95b/addons/mrp/models/mrp_production.py#L966-L967